scripts/prepdocs.py

```diff
@@ -35,6 +35,11 @@
 SENTENCE_SEARCH_LIMIT = 100
 SECTION_OVERLAP = 100
 
+OPEN_AI_TOKEN_CACHE = {}
```

Comment:
The capitalization makes this seem like a constant, but its a mutable dict, so I think it should just be lowercase. 
https://pep8.org/#constants